.circleci/config.yml

```diff
@@ -18,7 +18,7 @@ jobs:
       - checkout
 
       - setup_remote_docker:
-          version: 20.10.18
+          version: docker24
```

Comment:
docker 20 deprecated and will start experiencing scheduled brownouts in february https://discuss.circleci.com/t/remote-docker-image-deprecations-and-eol-for-2024/50176